development/build/index.js

```diff
@@ -127,6 +127,8 @@ async function defineAndRunBuildTasks() {
         'Promise',
         'JSON',
         'Date',
+        'Proxy',
```

Comment:
This is a notable change that should be called out in the PR description at the very least, since it expands the available global APIs

Reply:
mmm I am not sure the implications of this change :thinking: let's see if @legobeat can add a bit more context, so we cna merge the PR. Thank you both :pray:

Reply:
Is it enough to add something like this to the PR description?

```
Update LavaMoat configuration to add `Proxy` to the list of globals
```

(I'm not sure how exactly to characterize this change either, I just know that it is necessary as I ran into the same issue when attempting to update Chromedriver.)